packages/api/src/modules/user/user.service.ts

```diff
@@ -531,15 +530,16 @@ export class UserService {
         return roles.every(role => this.isRoleValid(role));
     }
 
-    private validEmail(email: string): UserServiceError | { success: true } {
+    private async validEmail(email: string): Promise<UserServiceError | { success: true }> {
         if (!REGEX_MAIL.test(email)) {
             return { success: false, message: "Email is not valid", code: UserServiceErrors.CREATE_INVALID_EMAIL };
         }
 
-        if (!ACCEPTED_EMAIL_DOMAIN.some(domain => email.endsWith(domain))) {
+        const emailDomains = await emailDomainsService.getAll();
```

Comment:
Je pense que la logique de validation du domaine de l'email devrai etre faire dans le service emailDomains

Reply:
Le service pourrais exposer un methode genre isAcceptedEmailDomain

Reply:
Sauf si on passe tous dans le configRepo dans ce cas la on peut laisser cette logique dans le user.service

Reply:
Je change pour mettre dans emailDomains.service pour le moment. Une réflexion devra être faite pour la gestion de ce genre de données liés à l'administration (puisque actuellement ConfigurationEntity n'est qu'un tuple nom/valeur et correspond bien à des variables de configuration).

Reply:
ConfigurationEntity peut prendre en data un tableau, ça ne pause pas vraiment de soucis !
On lui passe le type de donnée que l'on souhaite !
Elle a été pensée pour être le lieu unique de la configuration de la plateforme !

Reply:
Ça me parle de mettre ça dans le module de config. Plus que dans user déjà, et si les tableaux sont effectivement bien gérés en BDD, c'est idéal en effet

Reply:
@VSerain @alice-telescoop je fais les modification de base dans cette PR ? -> passage dans configuration avec un seul tableau contenant tous les domaines ?

Reply:
Je dirais oui ! C'est pas le plus classe, mais ça va faire le taf !

Reply:
good to me

Reply:
done